development/imagebump-detector/pkg/common.go

```diff
@@ -0,0 +1,49 @@
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+
+	"gopkg.in/yaml.v3"
+)
+
+func ParseNotationFile(filePath string) (string, string, error) {
+	f, err := os.Open(filePath)
+	defer f.Close()
+	if err != nil {
+		panic(err)
+	}
+	r := bufio.NewReader(f)
+	for {
+		line, err2 := r.ReadString('\n')
+		if err2 == io.EOF {
```

Comment:
If you don't need to access the `err` above then you should override this name in the scope of the loop.
```suggestion
		line, err2 := r.ReadString('\n')
		if err2 == io.EOF {
```
```suggestion
		line, err := r.ReadString('\n')
		if err == io.EOF {
```